tests/acceptance/render-tree-test.js

```diff
@@ -114,24 +114,6 @@ module('Render Tree Tab', function(hooks) {
     assert.dom(rows[0].querySelector('.js-render-profile-name')).hasText('First View Rendering');
     assert.dom(rows[1].querySelector('.js-render-profile-name')).hasText('Second View Rendering');
 
-    await fillIn('.js-render-profiles-search input', 'first');
```

Comment:
@nummi do we not need these tests anymore?

Reply:
Search filtering is still tested in the lines below these — the code removed here was testing the (broken, see GIF) functionality of opening of children rows.